observability/genaisdk/main.py

```diff
@@ -0,0 +1,62 @@
+import logging
+
+from dotenv import load_dotenv
+from google.genai import Client
+from otel_setup import setup_otel_instrumentation, setup_otel_to_gcp_wiring
+
+
+# [START main_logic_support_snippet]
+def setup_telemetry() -> None:
+    """Initializes Open Telemetry instrumentation and wiring.
+
+    This ensures that:
+
+      1. The Gen AI SDK and its relevant dependencies collect and
+         emit telemetry data to the Open Telemetry API (library).
+
+      2. The Open Telemetry library is configured and initialized
+         so that the API is wired up with the Google Cloud Observability
+         suite rather than the default no-op implementation.
+    """
+    setup_otel_to_gcp_wiring()
+    setup_otel_instrumentation()
```

Comment:
Is this helper method really needed?

Reply:
It is intended to help develop the mental model of what "setting up telemetry" entails.